aea/cli/core.py

```diff
@@ -193,6 +197,100 @@ def add_key(ctx: Context, type_, file):
     )
 
 
+@cli.command()
+@click.argument(
+    "type_",
+    metavar="TYPE",
+    type=click.Choice(
+        [DefaultCrypto.identifier, FetchAICrypto.identifier, EthereumCrypto.identifier]
+    ),
+    required=True,
+)
+@pass_ctx
+def get_address(ctx: Context, type_):
+    """Get the address associated with the private key."""
+    try_to_load_agent_config(ctx)
+
+    _verify_or_create_private_keys(ctx)
+    private_key_paths = dict(
+        [
+            (identifier, config.path)
+            for identifier, config in ctx.agent_config.private_key_paths.read_all()
+        ]
+    )
+    try:
+        wallet = Wallet(private_key_paths)
+        address = wallet.addresses[type_]
+        logger.info("{}".format(address))
+    except ValueError as e:  # pragma: no cover
+        logger.error(str(e))  # pragma: no cover
+
+
+@cli.command()
+@click.argument(
+    "type_",
+    metavar="TYPE",
+    type=click.Choice([FetchAICrypto.identifier, EthereumCrypto.identifier]),
+    required=True,
+)
+@pass_ctx
+def get_wealth(ctx: Context, type_):
```

Comment:
When will we support different addresses on the same network at the same time? So instead of `type_` as a choice, they should be among user-populated identifiers (e.g. `fetchai_1`, `fetchai_test` etc.), which are of type `fetchai`.

Reply:
We can start looking into this when it is a user request or we need it for our demos. For now I consider that YAGNI.

Reply:
Note, I see two cases:
- different node addresses on the same network
- different networks of the same type (e.g. Ropsten and Goerli and Mainnet)

Reply:
Feel free to open an issue @marcofavorito if you think this is important. I don't think so for the time being.